app/models/like.rb

```diff
@@ -1,3 +1,9 @@
 class Like < ApplicationRecord
-  belongs_to :post, class_name: 'Post', foreign_key: 'author_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'user_id'
+  belongs_to :user, class_name: 'User', foreign_key: 'post_id'
```

Comment:
- [ ] You interchanged the position of the foreign keys here. The post should have `post_id`, not `user_id`. And vice versa for the user.

- [ ] **[OPTIONAL]** Since there are models called Post and User, you don't really need this much verbose to declare the association. You can remove `class_name` and `foreign_key`.

  ```
  belongs_to :post
  belongs_to :user
  ```